src/main/java/com/varukha/onlinebookstore/service/impl/BookServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.varukha.onlinebookstore.service.impl;
+
+import com.varukha.onlinebookstore.dto.BookDto;
+import com.varukha.onlinebookstore.dto.CreateBookRequestDto;
+import com.varukha.onlinebookstore.mapper.BookMapper;
+import com.varukha.onlinebookstore.model.Book;
+import com.varukha.onlinebookstore.repository.BookRepository;
+import com.varukha.onlinebookstore.service.BookService;
+import jakarta.persistence.EntityNotFoundException;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class BookServiceImpl implements BookService {
+    private final BookRepository bookRepository;
+    private final BookMapper bookMapper;
+
+    public BookServiceImpl(BookRepository bookRepository, BookMapper bookMapper) {
+        this.bookRepository = bookRepository;
+        this.bookMapper = bookMapper;
```

Comment:
We can use @RequiredArgsConstructor to automaticaly create and initialize all field